### ISSUE TITLE
i need to calculate the vertices of the polygon how can i get that after applying the margin to the polygon

### ISSUE THREAD

**Issue by thamee** [w8r/polygon-offset#20]
if the polygon has four vertices, after performing margin function the output is more than four points , but in some scenarios, I need to calculate the vertices I can I achieve that
![image](https://user-images.githubusercontent.com/29250767/74713078-d05df780-524d-11ea-9a52-311ec4e0a4af.png)

**Comment by w8r** [comment]
is your geometric shape rectangular?

**Comment by thamee** [comment]
> is your geometric shape rectangular?
yes I have four points of the vertices and i know the height and width of the rectangle before pass into the offset function but I need the vertices after the margin function
